app/helpers/application_helper/button/vm_instance_scan.rb

```diff
@@ -3,14 +3,12 @@ class ApplicationHelper::Button::VmInstanceScan < ApplicationHelper::Button::Bas
 
   def skip?
     return false if @record.kind_of?(OrchestrationStack) && @display == "instances"
-    !(@record.is_available?(:smartstate_analysis) ||
-      @record.is_available_now_error_message(:smartstate_analysis)) ||
-    !@record.has_proxy?
+    @record.archived? || !@record.has_proxy?
   end
 
   def disable?
```

Comment:
I think it relies on whether error message is returned or not. If error message is not returned, then the button is not disabled, but if error message is returned, then it is disabled and that particular error message is shown on hover.